packages/manager/modules/pci/src/projects/project/ssh-keys/remove/remove.controller.js

```diff
@@ -0,0 +1,74 @@
+const MESSAGES_CONTAINER_NAME = 'pci.projects.project.sshKeys';
+
+export default class ProjectSshKeysControllerRemove {
+  /* @ngInject */
+  constructor($state, $stateParams, $translate, CucCloudMessage, OvhApiCloudProjectSshKey) {
+    this.$state = $state;
+    this.$translate = $translate;
+    this.CucCloudMessage = CucCloudMessage;
+    this.OvhApiCloudProjectSshKey = OvhApiCloudProjectSshKey;
+
+    this.serviceName = $stateParams.projectId;
+    this.keyId = $stateParams.keyId;
+    this.isLoading = false;
+
+    this.$onInit();
```

Comment:
Is it needed to call the `$onInit` hook inside the constructor class? Shouldn't be the controller's instantiation which trigger this hook?

Reply:
Only for component, will do it in next PR ;)